src/main/include/utils/Logging.h

```diff
@@ -3,36 +3,44 @@
 
 #include <frc/smartdashboard/SmartDashboard.h>
 
-#include <format>
 #include <iostream>
 
 namespace Logging {
+const std::string keyPrefix = "LOG";
 enum class Level { VERBOSE, INFO, WARNING, ERROR };
 
 static void logToSmartDashboard(const std::string& key, const std::string& msg,
-                                Level level) {
-    frc::SmartDashboard::PutString(key, getLevelString(msg, level));
+                                Level level = Level::INFO) {
+    frc::SmartDashboard::PutString(key, msg);
 }
 
 static void logToStdOut(const std::string& key, const std::string& msg,
```

Comment:
Add an if that checks if level is above the min log level to hide unwanted messages